### PR TITLE
Title and Description Component

### PR DESCRIPTION
**Pull request by DiegoRuiz454**
# Description

**This is the title and description component that can be used for these several scenarios**

Fixes https://github.com/HackYourFuture-CPH/rate-my-cv/issues/88

# How to test?

ican be tested in http://localhost:3000/
in the storybook as title component

# Checklist

- [☑] I have performed a self-review of my own code
- [☑] I have commented my code, particularly in hard-to-understand areas
- [☑] I have made corresponding changes to the documentation
- [☑] This PR is ready to be merged and not breaking any other functionality
title description component

### CODE REVIEW

File: src/client/components/Title/TitleDesc.styles.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  .description{ 
+      font-family: 'Times New Roman', Times, serif;
+      padding-top: 1em;
+      font-style: normal;
+      font-size: 18px;
+      font-weight: normal;
+      line-height: 47px;
+      margin: 1rem;
+      
+    }
```
Comment (arya56) [inline src/client/components/Title/TitleDesc.styles.css:20]: probably some left padding for the whole component?

Comment (DiegoRuiz454) [reply src/client/components/Title/TitleDesc.styles.css:20]: sure, good idea! This component is created with the intention that the developer who wants to implement it in another component can make modifications to the design. as the width, padding, margin etc..

File: src/client/components/Title/TitleDesc.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+const TitleDesc = ({title,description}) =>{ return (<div> <h3 className="title">{title}</h3>
+                                           <p className="description">{description}</p></div> ); };
+
+export default TitleDesc
+
+TitleDesc.propTypes = {
+title: PropTypes.string.isRequired,
+description: PropTypes.string.isRequired,
+};
```
Comment (arya56) [inline src/client/components/Title/TitleDesc.js:13]: Proptypes should be assigned before export. 
```suggestion
TitleDesc.propTypes = {
  title: PropTypes.string.isRequired,
  description: PropTypes.string.isRequired,
};

export default TitleDesc
```

File: src/client/components/Title/TitleDesc.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './TitleDesc.styles.css'
+
+const TitleDesc = ({title,description}) =>{ return (<div> <h3 className="title">{title}</h3>
+<p className="description">{description}</p></div> ); };
+
```
Comment (bytewiz) [inline src/client/components/Title/TitleDesc.js:7]: The indentation is a bit weird here

File: src/client/components/Title/TitleDesc.styles.css (hunk 1 above)
Comment (bytewiz) [reply src/client/components/Title/TitleDesc.styles.css:20]: Yea we should not and spacing inside the components if possible :)

File: src/client/components/Title/TitleDesc.js (hunk 3 above)
Comment (DiegoRuiz454) [reply src/client/components/Title/TitleDesc.js:7]: ?

Comment (KavyaReddy732) [reply src/client/components/Title/TitleDesc.js:7]: the code is compressed. use a line for every tag 
```suggestion
const TitleDescription = ({title,description}) =>{
 return (
 <div> 
 <h3 className="title">{title}</h3>
<p className="description">{description}</p>
</div> )
 };

```

File: src/client/components/Title/TitleDesc.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+        
+
+
+
+TitleDesc.propTypes = {
+title: PropTypes.string.isRequired,
+description: PropTypes.string.isRequired,
+};
+
+export default TitleDesc
```
Comment (valmdz) [inline src/client/components/Title/TitleDesc.js:22]: The indentation should be more like this, it makes it easier to read: 

```
import React from 'react';
import PropTypes from 'prop-types';
import './TitleDesc.styles.css';

const TitleDesc = ({ title, description }) => {
  return (
    <div>
      <h3 className="title">{title}</h3>
      <p className="description">{description}</p>
    </div>
  );
};

TitleDesc.propTypes = {
  title: PropTypes.string.isRequired,
  description: PropTypes.string.isRequired,
};

export default TitleDesc;

```

Comment (valmdz) [reply src/client/components/Title/TitleDesc.js:22]: You can set a linter for vscode so that it helps to format the files ;)

File: src/client/components/Title/TitleDesc.stories.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+import  TitleDesc  from './TitleDesc';
+
+
+export default { title: 'Title component' };
+
+export const title = () => <TitleDesc title="Here goes the title" description="Heres goes the description" />;
```
Comment (valmdz) [inline src/client/components/Title/TitleDesc.stories.js:8]: In this case, it should be more like this:

```
import React from 'react';

import TitleDesc from './TitleDesc';

export default { title: 'Title component' };

export const title = () => (
  <TitleDesc
    title="Here goes the title"
    description="Heres goes the description"
  />
);

```

Comment (DiegoRuiz454) [reply src/client/components/Title/TitleDesc.stories.js:8]: Thank you valentina, i think you did a good review. in my concept when someone as reviewer suggest a possible solution this is the best way as a teamwork helping each other, instead of say this is weird, or something like that.

File: src/client/components/Title/TitleDesc.styles.css [hunk 6]
```diff
@@ -0,0 +1,20 @@
+.title{
+    font-family: 'Times New Roman', Times, serif;
```
[the remark this answers is not in the document]
Comment (arya56) [reply src/client/components/Title/TitleDesc.styles.css:2]: Great eyes @valmdz :smiley:

File: src/client/components/Title/TitleDesc.js [hunk 7]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './TitleDesc.styles.css'
+
+const TitleDesc = ({title,description}) =>{
```
Comment (pdsorensen) [inline src/client/components/Title/TitleDesc.js:5]: Fix indentation and styling please. You can set this up in VS Code (if you don't know how reach out to your classmates),

File: src/client/components/Title/TitleDesc.styles.css [hunk 8]
```diff
@@ -0,0 +1,18 @@
+.title{
+      font-family: Roboto;
```
Comment (pdsorensen) [inline src/client/components/Title/TitleDesc.styles.css:2]: I'm seeing alot of font definitions in css? What is the default font-family, and why do we need to override the defaults? Same question regarding `font-style`.

File: src/client/components/Title/TitleDesc.styles.css [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+      font-style: normal;
+      font-size: 25px;
+      font-weight: bold;
+      line-height: 47px;
+      color: rgb(14, 13, 13);
+      margin-bottom: 0rem;
+      }
```
Comment (pdsorensen) [inline src/client/components/Title/TitleDesc.styles.css:9]: Let's fix spacing and indentation.

**Review by pdsorensen — CHANGES_REQUESTED** [review]
Minor changes. :)

File: src/client/components/Title/TitleDesc.styles.css [hunk 10]
```diff
@@ -0,0 +1,18 @@
+.title{
+      font-family: Roboto;
+      font-style: normal;
```
Comment (valmdz) [inline src/client/components/Title/TitleDesc.styles.css:3]:
```suggestion

```

File: src/client/components/Title/TitleDesc.styles.css (hunk 8 above)
Comment (valmdz) [inline src/client/components/Title/TitleDesc.styles.css:2]:
```suggestion
```

File: src/client/components/Title/TitleDesc.styles.css [hunk 11]
```diff
@@ -0,0 +1,18 @@
...
+      font-size: 25px;
+      font-weight: bold;
+      line-height: 47px;
+      color: rgb(14, 13, 13);
+      margin-bottom: 0rem;
+      }
+    ​
+.description{ 
+      font-family: Roboto;
```
Comment (valmdz) [inline src/client/components/Title/TitleDesc.styles.css:12]:
```suggestion

```

**Review by dpfernandes — DISMISSED** [review]
Please remove unrelated changes on App.js and Footer.js

**Comment by DiegoRuiz454** [comment]
> Please remove unrelated changes on App.js and Footer.js

ok i'll do thanks

File: src/client/components/Title/TitleDesc.styles.css [hunk 12]
```diff
@@ -0,0 +1,18 @@
+.container {
+  font-family: Roboto;
```
Comment (pdsorensen) [inline src/client/components/Title/TitleDesc.styles.css:2]: Why is this needed?

File: src/client/components/Title/TitleDesc.styles.css [hunk 13]
```diff
@@ -0,0 +1,18 @@
...
+}
+
+.title {
+  font-style: normal;
```
Comment (pdsorensen) [inline src/client/components/Title/TitleDesc.styles.css:6]: Why is this needed?

File: src/client/components/Title/TitleDesc.styles.css [hunk 14]
```diff
@@ -0,0 +1,18 @@
...
+  font-size: 25px;
+  font-weight: bold;
+  line-height: 47px;
+  color: rgb(14, 13, 13);
+  margin-bottom: 0rem;
+}
+​ .description {
+  font-size: 18px;
+  font-weight: normal;
```
Comment (pdsorensen) [inline src/client/components/Title/TitleDesc.styles.css:15]: Why is this needed?

**Review by pdsorensen — CHANGES_REQUESTED** [review]

File: src/client/components/Title/TitleDesc.styles.css [hunk 15]
```diff
@@ -0,0 +1,4 @@
+  .container{
```
Comment (pdsorensen) [inline src/client/components/Title/TitleDesc.styles.css:1]: I think this class has been added alot of other places, and that can be very hard to debug. Can you name the class something more specific, e.g `title-desc-container`. 

Also, i'm not sure why we need absolute position?

**Review by pdsorensen — CHANGES_REQUESTED** [review]
Keep going @DiegoRuiz454 🙌

**Review by pdsorensen — APPROVED** [review]
